src/libraries/System.Formats.Tar/src/Resources/Strings.resx

```diff
@@ -0,0 +1,252 @@
+  </data>
+  <data name="TarUnexpectedMetadataEntry" xml:space="preserve">
+    <value>A metadata entry of type '{0}' was unexpectedly found after a metadata entry of type '{1}'.</value>
+  </data>
+  <data name="TarEntriesInDifferentFormats" xml:space="preserve">
+    <value>The archive contains entries in different formats.</value>
+  </data>
+  <data name="TarEntryBlockOrCharacterExpected" xml:space="preserve">
+    <value>Cannot set the DevMajor or DevMinor fields on an entry that does not represent a block or character device.</value>
+  </data>
+  <data name="TarEntryHardLinkOrSymLinkExpected" xml:space="preserve">
+    <value>Cannot set the LinkName field on an entry that does not represent a hard link or a symbolic link.</value>
+  </data>
+  <data name="TarEntryModeOutOfRange" xml:space="preserve">
+    <value>The mode must be a base 10 number between 0 and 511 (777 in octal).</value>
+  </data>
+  <data name="TarEntryNotARegularFile" xml:space="preserve">
+    <value>Entry type is not a regular file, so it does not support setting the data stream.</value>
+  </data>
+  <data name="TarEntryTypeNotSupported" xml:space="preserve">
+    <value>Entry type '{0}' not supported in format '{1}'.</value>
+  </data>
+  <data name="TarEntryTypeNotSupportedForExtracting" xml:space="preserve">
+    <value>Entry type not supported for extraction: '{0}'</value>
+  </data>
+  <data name="TarExtractingResultsInOutside" xml:space="preserve">
+    <value>Extracting Tar entry would have resulted in a file outside the specified destination directory.</value>
+  </data>
+  <data name="TarGnuFormatExpected" xml:space="preserve">
+    <value>A GNU format was expected, but could not be reliably determined for entry '{0}'.</value>
+  </data>
+  <data name="TarMalformedTwoExtendedAttributesEntries" xml:space="preserve">
+    <value>The archive is malformed. It contains two extended attributes entries in a row.</value>
+  </data>
+  <data name="TarPaxFormatExpected" xml:space="preserve">
+    <value>A PAX format was expected, but could not be reliably determined for entry '{0}'.</value>
+  </data>
+  <data name="TarPosixFormatExpected" xml:space="preserve">
+    <value>A POSIX format was expected (Ustar or PAX), but could not be reliably determined for entry '{0}'.</value>
+  </data>
+  <data name="TarSizeFieldNegative" xml:space="preserve">
+    <value>The size field is negative in the tar entry '{0}'.</value>
+  </data>
+  <data name="TarSizeFieldTooLargeForExtendedAttribute" xml:space="preserve">
+    <value>The value of the size field for the current entry of type '{0}' is beyond the expected length.</value>
+  </data>
+  <data name="TarTooManyGlobalExtendedAttributesEntries" xml:space="preserve">
+    <value>The archive has more than one global extended attributes entry.</value>
+  </data>
+  <data name="TarUnsupportedFile" xml:space="preserve">
+    <value>The file '{0}' is not supported for tar archiving.</value>
```

Comment:
```suggestion
    <value>The file '{0}' is a type of file not supported for tar archiving.</value>
```
? 
Incidentally I was wondering whther this would ever appear on Windows, but I see it would for files with Hidden or System attributes - what's the reasoning there - do we have precedent for ignoring such files?

Reply:
This code isn't necessarily ignoring `FileAttributes.Hidden` or `FileAttributes.System`, The `attributes` variable holds enum flags, so I would not expect to find those values by themselves.

Reply:
No change here for now.

I added to my todo list a test that verifies that a hidden file can be read to create an entry.